docs/user/configuring-agent.md

```
@@ -135,6 +136,115 @@
+
+> [!WARNING]
+> Setting `auth-enabled: true` uses a well-known password to take TPM ownership. 
+
+#### Enrollment Failures
+
+If enrollment fails with TPM-related errors:
+
+1. Check that TPM CA certificates are installed on the server (see [TPM Device Authentication](tpm-authentication.md#configuring-tpm-ca-certificates-in-flight-control-service))
+
+2. Verify the agent can generate CSR:
+   ```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)